src/app/cmdoptions.cpp

```diff
@@ -515,55 +515,70 @@ QString makeUsage(const QString &prgName)
-                                   "'parameter-name', environment variable name is 'QBT_PARAMETER_NAME' (in upper "
-                                   "case, '-' replaced with '_'). To pass flag values, set the variable to '1' or "
-                                   "'TRUE'. For example, to disable the splash screen: "), 0) << "\n"
-           << QLatin1String("QBT_NO_SPLASH=1 ") << prgName << '\n'
-           << wrapText(QObject::tr("Command line parameters take precedence over environment variables"), 0) << '\n';
+        << PROFILE_OPTION.usage(QObject::tr("path"))
+            << wrapText(QObject::tr("Store configuration files in <path> (portable mode)")) << '\n'
+        << RELATIVE_FASTRESUME.usage()
+            << wrapText(QObject::tr("Make qBittorrent store paths of torrents' files "
+                                    "relative to the profile directory")) << '\n'
+        << WEBUI_PORT_OPTION.usage(QObject::tr("port"))
+            << wrapText(QObject::tr("Change the Web UI port (default: 8080)")) << '\n'
+        << '\n'
+        
+        << Option::padUsageText(QObject::tr("URI (files, magnets, ...)"))
+            << wrapText(QObject::tr("Download the torrents passed by the user")) << '\n'
+        << '\n'
+
+        << wrapText(QObject::tr("Options when adding new torrents:"), 0) << '\n'
+        << PAUSED_OPTION.usage()
+            << wrapText(QObject::tr("Add torrents as started or paused")) << '\n'
+        << CATEGORY_OPTION.usage(QObject::tr("name"))
+            << wrapText(QObject::tr("Assign torrents to category."
+                                    "If the category doesn't exist, it will be created.")) << '\n'
+        << FIRST_AND_LAST_OPTION.usage()
+            << wrapText(QObject::tr("Download first and last pieces first")) << '\n'
+        << SAVE_PATH_OPTION.usage(QObject::tr("path"))
+            << wrapText(QObject::tr("Save torrents to this path")) << '\n'
+        << SEQUENTIAL_OPTION.usage()
+            << wrapText(QObject::tr("Download torrent pieces in sequential order")) << '\n'
+        << SKIP_DIALOG_OPTION.usage()
+            << wrapText(QObject::tr("Specify whether the \"Add New Torrent\" dialog "
+                                    "opens when adding a torrent.")) << '\n'
+        << SKIP_HASH_CHECK_OPTION.usage()
+            << wrapText(QObject::tr("Skip hash check")) << '\n'
+        << '\n'
+
+        << wrapText(QObject::tr("Option values may be supplied via environment variables. "
+                                "For an option named 'parameter-name', "
+                                "its environment variable name is 'QBT_PARAMETER_NAME' "
+                                "(in upper case, '-' replaced with '_'). To pass flag values, "
+                                "set the variable to '1' or 'TRUE'. For example, "
+                                "to prevent added torrents from auto-starting: "), 0) << "\n"
```

Comment:
lets also clean this up: `"\n"` -> `'\n'`

Reply:
Fix